README.md

```diff
@@ -8,10 +8,10 @@ You can enter the example for information about  "How to run"
 
 ## Bizdemo
 
-- [hertz_gorm:](bizdemo/hertz_gorm) Example of using gorm in hertz server
-- [hertz_gorm_gen:](bizdemo/hertz_gorm_gen) Example of using gorm/gen & proto IDL in hertz server
-- [hertz_jwt:](bizdemo/hertz_jwt) Example of using jwt in hertz server
-- [hertz_session:](bizdemo/hertz_session) Example of using distributed session and csrf in hertz server
+- [bizdemo/hertz_gorm:](bizdemo/hertz_gorm) Example of using gorm in hertz server
```

Comment:
No need for this duplicated prefix because the contextual scope has been clearly marked on the title([line 9](https://github.com/cloudwego/hertz-examples/pull/71/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R9)).